packages/ses/src/compartment-evaluate.js

```diff
@@ -13,6 +13,30 @@ import {
 } from './transforms.js';
 import { performEval } from './evaluate.js';
 
+export const makeCompartmentLocalObject = (compartmentFields, options) => {
```

Comment:
We have a separate issue about renaming “local object” to “global lexicals” throughout to make this clearer. This makes more work for later and I wouldn’t mind if this change fixed that in a first commit. I also don’t mind doing more work later.